Draft.md

```diff
@@ -330,9 +372,12 @@ Schließlich ist es für Entwickler oft interessant die Software selbst mitzuges
 
 > [HM] Robustheit ist aus meiner Sicht eher, dass die Software nicht abstürzt, nicht in einer Endlosschleife hängen bleibt und keine Speicherleaks hat, die ggf. den ganzen Computer zu Fall bringen. Robustheit gegen Stromausfall ist eher Sache der Versorgung bzw. Aufgabe einer USV.
 
+> [MT] stimme HM zu. Anne hat da schon recht, dass es sich hier eher um ein Usability-Aspekt handelt. Ich würde das vielleicht sogar komplett streichen.
```

Comment:
Ja wir können das gerne als Usability Aspekt verstehen, der aber meiner Meinung nach sehr wichtig ist. Niemand mag gerne, dass seine Arbeit in der Software bei einem Absturz verloren geht. Wo sollten wir das denn am besten platzieren?

Reply:
Ich würde vorschlagen, dass wir unter `### Bedienbarkeit und Zielgruppenorientierung` einen neuen Punkt `#### Perfomanz und Robustheit` nach `#### Interface` anlegen, der dann diesen und den vorangegangenen Punkt aufnimmt.

Reply:
Habe ich so umgesetzt, müsste nur noch mal gegengelesen und die Kommentare dann aufgelöst werden.